provider/resource.go

```diff
@@ -2592,6 +2598,11 @@ func Provider() tfbridge.ProviderInfo {
 		azureResource(azureContainerService, "RegistryWebook"),
 		azureResource(azureContainerService, "RegistryWebhook"), azureContainerService, azureContainerService, nil)
 
+	// fix spelling of sentinel.AutomationRule
```

Comment:
I assume the rename works as advertised - would the misspelled rerource be removed in the next release? or are we providing for backwards compatibility here?